MIGRATION.md

```diff
@@ -2,6 +2,27 @@
 
 These guides below are provided to ease the transition of existing applications using the Onfido SDK from one version to another that introduces breaking API changes.
 
+## `3.0.1` -> Next Version
+
+- Set the CSS units used in the SDK to be relative (`em`) units. If you previously set the font-size of `.onfido-sdk-ui-Modal-inner`, it is recommended that you remove this `font-size` override.
```

Comment:
worth clarifying that the first sentence is not an instruction, yet a reference to an internal change

Reply:
Good point - done :)